lib/nix-interop/derivation.ncl

```diff
@@ -1,54 +1,12 @@
 let type_field = "$__organist_type" in
 
+let nix_string = import "./nix-string.ncl" in
+
 let predicate | doc "Various predicates used to define contracts"
-  = {
-    is_nix_path = fun x =>
-      std.is_record x
-      && std.record.has_field type_field x
-      && x."%{type_field}" == "nixPath",
-    is_nix_placeholder = fun x =>
-      std.is_record x
-      && std.record.has_field type_field x
-      && x."%{type_field}" == "nixPlaceholder",
-    is_nix_to_file = fun x =>
-      std.is_record x
-      && std.record.has_field type_field x
-      && x."%{type_field}" == "nixToFile",
-    is_nix_input = fun x =>
-      std.is_record x
-      && std.record.has_field type_field x
-      && x."%{type_field}" == "nixInput",
-    is_nix_string = fun value =>
-      std.is_record value
-      && std.record.has_field type_field value
-      && value."%{type_field}" == "nixString",
-    is_nickel_derivation = fun x =>
-      std.is_record x
-      && std.record.has_field type_field x
-      && x."%{type_field}" == "nickelDerivation",
-    is_derivation = fun x =>
-      is_nickel_derivation x
-      || is_nix_input x,
-    is_nix_call = fun value =>
-      std.is_record value
-      && std.record.has_field type_field value
-      && value."%{type_field}" == "callNix",
-    is_string_fragment = fun x =>
-      is_derivation x
-      || std.is_string x
-      || is_nix_path x
-      || is_nix_placeholder x
-      || is_nix_to_file x
-      || is_nix_call x
-  }
+  = {}
```

Comment:
Why do we keep an empty `predicates` here?

Reply:
Because I forgot to drop it? :sweat_smile: 

Removed in 2fd0bd3